rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -130,8 +129,6 @@ MasterGraph::MasterGraph(size_t batch_size, RocalAffinity affinity, int gpu_id,
 #endif        
         _first_run(true),
         _processing(false),
-        _internal_batch_size(compute_optimum_internal_batch_size(batch_size, affinity)),
```

Comment:
Do we not require to use batch_size anywhere in master_graph ?

Reply:
internal batch size is not required in master graph